src/frontend-common/cheevos.cpp

```diff
@@ -1101,6 +1110,13 @@ void UnlockAchievement(u32 achievement_id, bool add_notification /* = true*/)
     return;
   }
 
+  if (achievement->category != AchievementCategory::Core)
+  {
+    Log_WarningPrintf("Skipping sending achievement %u unlock to server because it's not from the core set.",
```

Comment:
Maybe it'd be worth including this somewhere in the unlock message? (so the user doesn't go looking online for it)

Reply:
So, append the information that it's an unofficial achievement somewhere in the unlock pop-up?

Reply:
Sure, that works